fxp2aupreset.py

```diff
@@ -173,12 +182,12 @@ def main():
     args = get_arguments(parser)
 
     # enumerate all the .fxp files in the current directory
-    os.chdir(args.path)
+    chdir(args.path)
 
-    fxpFileList = glob("*.fxp")
+    fxpFileList = findfiles('*.fxp',args.path)
```

Comment:
You can make `fnmatch.filter` do this as well:

```
fnmatch.filter(os.listdir(args.path), '*.[Ff][Xx][BbPp]')
```

This has the added advantage of matching both `.fxp` and `.fxb` files in both upper and lowercase.